cgrok/grokre.c

```diff
@@ -58,6 +58,7 @@ void grok_free(grok_t *grok) {
   grok_free_clone(grok);
   if (grok->patterns != NULL)
     tctreedel(grok->patterns);
+  free(grok);
```

Comment:
This method didn't do what the docs said, so the grok struct itself was never freed.